tests/oeedger8r/behavior/deepcopy_oddsize.edl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+  struct MyStruct {
+    [size=13] int* ptr;
+  };
+
+  trusted {
+    // This should error because the size parameter attached to
+    // `s.ptr` is not a multiple of the size of `int`.
+    public void deepcopy_oddsize([in, count=1] struct MyStruct* s);
```

Comment:
Ah ok. Are we doing the validation? If so can the validation be done at the struct declaration rather than the use?

Reply:
We are not (yet). It's a pending test that's not enabled, because we ought to validate, but don't, and Intel doesn't.